playbooks/roles/buildbot-master/files/master.cfg

```diff
@@ -0,0 +1,201 @@
+        util.ShellArg(command=['sudo', 'apt-get', 'install', '-y', 'python-demjson']),
+    ],
+))
+config_test_factory.addStep(steps.ShellSequence(
+    name='install dependencies via pip',
+    commands=[
+        util.ShellArg(command=['sudo', 'pip', 'install', '--allow-all-external', '-r', 'requirements.txt']),
+        util.ShellArg(command=['sudo', 'pip', 'install', '--allow-all-external', 'demjson']),
+    ],
+))
+config_test_factory.addStep(steps.ShellCommand(
+    name='test YAML files',
+    command=(
+        'for yml in $(find . -name "*.yml"); do\n'
+          'python -c "import sys,yaml; yaml.load(open(sys.argv[1]))" $yml >/dev/null\n'
+          'if [ $? -ne 0 ]; then\n'
+            'echo "ERROR parsing $yml"\n'
+            'exit 1\n'
+          'fi\n'
+        'done'
+    ),
+))
+config_test_factory.addStep(steps.ShellCommand(
+    name='lint JSON files with jsonlint',
+    command=(
+        'for json in $(find . -name "*.json"); do\n'
+          'jsonlint -v $json\n'
+          'if [ $? -ne 0 ]; then\n'
+            'echo "ERROR parsing $json"\n'
+            'exit 1\n'
+          'fi\n'
+        'done'
+    ),
+))
+config_test_factory.addStep(steps.ShellCommand(
+    name='jinja tests',
+    workdir='build/playbooks',
+    command=(
+        'for j2 in $(git diff --name-only refs/heads/master..$(git rev-parse --abbrev-ref HEAD) |grep -E \'.+\\.j2$|.+\\.yml\'); do\n'
+          'python ../tests/jinja_check.py ../$j2\n'
+          'if [ $? -ne 0 ]; then\n'
+            'echo "Jinja parsing error on $j2"\n'
+            'exit 1\n'
+          'fi\n'
+        'done'
+    ),
+))
+config_test_factory.addStep(steps.ShellSequence(
+    name='core tests',
+    commands=[
+        util.ShellArg(command=['set', '-e']),
+        util.ShellArg(command=['ROOT_DIR=$PWD']),
+        util.ShellArg(command=['cd', 'playbooks/edx-east']),
+        util.ShellArg(command=['ROLE_DIRS=$(/bin/ls -d roles/*)']),
+        util.ShellArg(
+            command=(
+                'cat <<EOF >travis-test.yml\n'
+                '- name: Play to test all roles\n'
+                '  hosts: all\n'
+                '  roles:\n'
+                'EOF'
+            )),
+        util.ShellArg(
+            command=(
+                'for role_dir in $ROLE_DIRS; do\n'
+                  'echo "    - $(basename $role_dir)" >> travis-test.yml\n'
+                'done'
+            )),
+        util.ShellArg(command=['ansible-playbook', '-i', 'localhost,', '--syntax-check', 'travis-test.yml']),
+        util.ShellArg(command=['output_dir="$PWD/test_output/env-dep"']),
+        util.ShellArg(command=['mkdir', '-p', '$output_dir']),
+        util.ShellArg(command=['ansible-playbook', '-i', 'localhost,', '-c', 'local', '--tags', 'edxapp_cfg', 'edxapp.yml' '-e' 'edxapp_user=`whoami`', '-e', 'edxapp_app_dir=$output_dir', '-e', 'edxapp_code_dir=$output_dir', '-e', 'EDXAPP_CFG_DIR=$output_dir']),
```

Comment:
I should probably put this on multiple lines..
This is also the command that appears to be failing, though I had to ssh in and run things manually to see the error message for some reason:
```
Traceback (most recent call last):
  File "/usr/local/bin/ansible-playbook", line 44, in <module>
    import ansible.playbook
ImportError: No module named ansible.playbook
```